src/alphaProxy.js

```diff
@@ -0,0 +1,38 @@
+const { createServer } = require('http');
+
+const alphaProxy = async (baseConfig, callback) => {
+  createServer((req, response) => {
```

Comment:
For security reasons, should this be restricted to localhost only?

Reply:
Yes, nice catch.